### ISSUE TITLE
Bug when swiping left from first image with infinite scroll

### ISSUE THREAD

**Issue by ndbroadbent** [stevenwanderski/bxslider-4#777]
Hello,

I've noticed there is a bug when swiping backwards on the first image in the gallery, if you have infinite scroll enabled. As soon as I release the touch event, the new image disappears. If I drag it a little bit and then release, it appears again. 

Please note that I have the control arrows enabled for web, and if I click the "previous" arrow, the last image is shown correctly. This only happens when swiping on mobile.

See the GIF for a demo:

![bxslider-bug](https://cloud.githubusercontent.com/assets/139536/6770236/5664d25e-d0e4-11e4-9d34-c7d44d99fefd.gif)

**Comment by ndbroadbent** [comment]
Looks like it has something to do with the `position` being negative before calling `el.goToPrevSlide();` in the `onTouchEnd` handler.

Here's my ugly workaround for now:

```
if (slider.active.index === 0)
	setPositionProperty(0, 'reset', 0);
el.goToPrevSlide();
```

**Comment by ndbroadbent** [comment]
Ignore that workaround ^

Have fixed the bug in #778

**Comment by Tidal-Wave** [comment]
Issue will be addressed in 4.2.4, The problem came from an experimental attempt to address an edge case in `setPositionProperty` that we will be removing.
